tox.ini

```diff
@@ -23,12 +23,19 @@ commands =
     twine check dist/*
 
 [testenv:release]
-basepython = python3
```

Comment:
Out of curiosity: why not require Python 3, and use `twine>=2`?

Reply:
I developed this process before twine 2, but I've been cutting releases on Python 3 for a long time. The expectation is that one will use Python 3; it's just not prescribed. I'm a little reluctant to set `basepython` because I'm not sure how well that's supported on Windows and it works fine without it. I'd rather defer to other metadata or package requirements to enforce the Python version if/when necessary.

Reply:
I would rather leave `basepython` here. I don't understand the FUD around how it works on Windows. It's basically the same as `tox -e py36` to my understanding

Reply:
In my environment, adding the `basepython=python3` seems to be the same as using `which python3`:

```
draft $ cat tox                                                                                                                                                           
cat: tox: No such file or directory
draft $ cat tox.ini                                                                                                                                                       
[tox]
skipsdist=True

[testenv]
skip_install=True
basepython=python3
commands=python -V
draft $ tox -e py36                                                                                                                                                       
/Users/jaraco/.local/pipx/venvs/tox/lib/python3.8/site-packages/tox/config/__init__.py:578: UserWarning: conflicting basepython version (set 3.8, should be 3.6) for env 'py36';resolve conflict or set ignore_basepython_conflict
  warnings.warn(
py36 run-test-pre: PYTHONHASHSEED='1648128078'
py36 run-test: commands[0] | python -V
Python 3.8.0
________________________________________________________________________________ summary _________________________________________________________________________________
  py36: commands succeeded
  congratulations :)

draft $ tox                                                                                                                                                               
python run-test-pre: PYTHONHASHSEED='4138533541'
python run-test: commands[0] | python -V
Python 3.8.0
________________________________________________________________________________ summary _________________________________________________________________________________
  python: commands succeeded
  congratulations :)
draft $ which python3                                                                                                                                                     
/Library/Frameworks/Python.framework/Versions/3.8/bin/python3
```

That suggests to me that if a `python3` binary didn't exist, it may not work. On Windows, `python3` isn't a binary, so I'm not sure what would happen there, and I'm not really interested in firing up a Windows VM to validate.

I guess I can add the entry, but what is the rationale? And at what point is the setting removed? And should this pattern be recommended for every project or just twine?